extensions/-SIPC-/time.js

```diff
@@ -387,16 +387,17 @@
       const totalSeconds = Scratch.Cast.toNumber(args.VALUE);
       const seconds =
         args.ROUND === "rounded"
-          ? Math.round(totalSeconds % 60)
-              .toString()
-              .padStart(2, "0")
+          ? Scratch.Cast.toString(Math.round(totalSeconds % 60)).padStart(
```

Comment:
Changing toString() to Cast.toString() here has no value and if anything harms readability. You already know Math.round() is returning a number. Hope this isn't how the rest of the pull request looks.

Reply:
Conceptually Scratch.Cast would be where any quirks in the anything-to-string conversion in Scratch-the-programming-language would live though it happens to be exactly the same as the default toString(). It doesn't need to be the same though.

Here, and many other places, you aren't dealing with a value from a Scratch project but rather some intermediate value you already know is a number. It's not benefiting at all to use Cast.